### ISSUE TITLE
[test-triage] rom_e2e_sigverify_always failing

### ISSUE THREAD

**Issue by johngt** [lowRISC/opentitan#17119]
### Hierarchy of regression failure

Block level

### Failure Description

No UVM error reported.
Intermittent failures since 11 January (Wednesday) but now all the rom_e2e_sigverify_always_* tests are failing.

### Steps to Reproduce

- GitHub Revision: [1e0c9c1a7](https://github.com/lowrisc/opentitan/tree/1e0c9c1a7da3e5793249a6dc55ccdbefaa6153d1)
- dvsim invocation command to reproduce the failure, inclusive of build and run seeds:
  ./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i rom_e2e_sigverify_always --build-seed 3470932822 --fixed-seed <NA> --waves fsdb
Note fixed seed above unavailable

### Tests with similar or related failures

- [ ] rom_e2e_sigverify_always_a_bad_b_bad_test_unlocked0
- [ ] rom_e2e_sigverify_always_a_bad_b_bad_dev
- [ ] rom_e2e_sigverify_always_a_bad_b_bad_prod
- [ ] rom_e2e_sigverify_always_a_bad_b_bad_prod_end
- [ ] rom_e2e_sigverify_always_a_bad_b_bad_rma
- [ ] rom_e2e_sigverify_always_a_bad_b_nothing_test_unlocked0
- [ ] rom_e2e_sigverify_always_a_bad_b_nothing_dev
- [ ] rom_e2e_sigverify_always_a_bad_b_nothing_prod
- [ ] rom_e2e_sigverify_always_a_bad_b_nothing_prod_end
- [ ] rom_e2e_sigverify_always_a_bad_b_nothing_rma
- [ ] rom_e2e_sigverify_always_a_nothing_b_bad_test_unlocked0
- [ ] rom_e2e_sigverify_always_a_nothing_b_bad_dev
- [ ] rom_e2e_sigverify_always_a_nothing_b_bad_prod
- [ ] rom_e2e_sigverify_always_a_nothing_b_bad_prod_end
- [ ] rom_e2e_sigverify_always_a_nothing_b_bad_rma

**Comment by johngt** [comment]
@engdoreis - best to have a look in general what is happening here and see if there is anything can be gained outside of the reports. Otherwise we may need to reassign to ROM team.

**Comment by johngt** [comment]
@engdoreis https://github.com/lowRISC/opentitan/issues/14634 - might be useful to look at for this also.

**Comment by engdoreis** [comment]
I tried to build locally with bazel first and I got an error related to the bitstream, but I'm not sure whether this relates to the regression error.
```
bazel test //sw/device/silicon_creator/rom/e2e:sigverify_always
```
```
WARNING:root:Closest bitstream to HEAD is f5ba9029609a01b3bcbe300b967ac547bce7f21e.
ERROR:root:Could not find the bitstreams to generate a BUILD file:'BUILD.bazel'
in files_by_extension:defaultdict(<class 'set'>, {'mmi': {'cache/f5ba9029609a01b3bcbe300b967ac547bce7f21e/otp.mmi', 'cache/f5ba9029609a01b3bcbe300b967ac547bce7f21e/rom.mmi', 'cache/f5ba9029609a01b3bcbe300b967ac547bce7f21e/chip_earlgrey_cw310_hyperdebug/otp.mmi', 'cache/f5ba9029609a01b3bcbe300b967ac547bce7f21e/chip_earlgrey_cw310_hyperdebug/rom.mmi'}, 'bit': {'cache/f5ba9029609a01b3bcbe300b967ac547bce7f21e/chip_earlgrey_cw310_hyperdebug/lowrisc_systems_chip_earlgrey_cw310_hyperdebug_0.1.bit'}})
using key:'f5ba9029609a01b3bcbe300b967ac547bce7f21e'
INFO: Repository bitstreams instantiated at:
  /home/doreis/lowrisc/git/opentitan/WORKSPACE:101:16: in <toplevel>
Repository rule bitstreams_repo defined at:
  /home/doreis/lowrisc/git/opentitan/rules/bitstreams.bzl:52:34: in <toplevel>
ERROR: An error occurred during the fetch of repository 'bitstreams':
   Traceback (most recent call last):
        File "/home/doreis/lowrisc/git/opentitan/rules/bitstreams.bzl", line 25, column 13, in _bitstreams_repo_impl
                fail("Bitstream cache not initialized properly.")
Error in fail: Bitstream cache not initialized properly.
ERROR: /home/doreis/lowrisc/git/opentitan/WORKSPACE:101:16: fetching bitstreams_repo rule //external:bitstreams: Traceback (most recent call last):
        File "/home/doreis/lowrisc/git/opentitan/rules/bitstreams.bzl", line 25, column 13, in _bitstreams_repo_impl
                fail("Bitstream cache not initialized properly.")
Error in fail: Bitstream cache not initialized properly.
ERROR: /home/doreis/lowrisc/git/opentitan/hw/bitstream/BUILD:91:10: //hw/bitstream:otp_mmi depends on @bitstreams//:otp_mmi in repository @bitstreams which failed to fetch. no such package '@bitstreams//': Bitstream cache not initialized properly.
ERROR: Analysis of target '//sw/device/silicon_creator/rom/e2e:sigverify_always_dev_a_bad_b_bad_fpga_cw310_rom' failed; build aborted: 
INFO: Elapsed time: 1.808s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
    Fetching @lowrisc_rv32imcb_files; fetching
    Fetching @bindgen_clang_linux; fetching
    Fetching ...d1/external/lowrisc_rv32imcb_files; Extracting /home/doreis/.cache/bazel/_bazel_doreis/71b02480bfcfe3a0741719bf45ac42d1/external/lowrisc_rv32imcb_files/temp5476790821708988334/lowrisc-toolchain-rv32imcb-20220524-1.tar.xz
    Fetching ...xternal/bindgen_clang_linux; Extracting /home/doreis/.cache/bazel/_bazel_doreis/71b02480bfcfe3a0741719bf45ac42d1/external/bindgen_clang_linux/temp8857233893462858725/clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz
```
I'll keep looking.

**Comment by engdoreis** [comment]
Fixed by PR #17198
